packages/9-public/@prisma/orm-postgres/README.md

```
@@ -0,0 +1,56 @@
+# @prisma/orm-postgres
+
+The one package a PostgreSQL application installs. It wires the framework, the SQL family, and the Postgres target into a single lazy client, and brings the rest of the stack along as exact-pinned dependencies:
+
+```jsonc
+// package.json
+{ "dependencies": { "@prisma/orm-postgres": "0.16.0" } }
+```
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)